### PR TITLE
fix: incorrect pipeline logs

### LINKED ISSUE #1525: Incorrect logs was printed by check_cluster_status.sh in e2e-TEST workflow
Check the actions logs in [pr-test.yaml](https://github.com/kubesphere/ks-installer/actions/workflows/pr-test.yml). 
1. Both the "Deploy KubeSphere to Kind" and "Test Plugin" steps printed the same logs. But the "Test Plugin" step should enable more components.
2.  There's are always string "CONTRIBUTING.md Dockerfile Dockerfile.complete LICENSE Makefile OWNERS README.md README_zh.md controller deploy docs env playbooks roles scripts " printed after each line.

/kind bug

### PR DESCRIPTION
**Pull request by RolandMa1986**
Signed-off-by: Roland.Ma <rolandma@yunify.com>
Fixes #1525 
1. Use self-host runner to load the e2e test, we can avoid resource limitation issues.
2. Fixes issue found in #1525.

### CODE REVIEW

**Comment by zackzhangkai** [comment]
@RolandMa1986  If we use self-hosted runner, maybe we no need to test devops plugin stand alone, as well as servicemesh.

**Comment by RolandMa1986** [comment]
> @RolandMa1986 If we use self-hosted runner, maybe we no need to test devops plugin stand alone, as well as servicemesh.

@zackzhangkai, Merged all plugins in one step after changing of some resources request.

File: .github/workflows/kind/kind.yaml [hunk 1]
```diff
@@ -6,6 +6,8 @@ nodes:
   extraMounts:
   - hostPath: /etc/localtime
     containerPath: /etc/localtime
+  - containerPath: /var/lib/kubelet/config.json
```
Comment (zryfish) [inline .github/workflows/kind/kind.yaml:9]: What are these mounts for?

File: .github/workflows/pr-test.yml [hunk 2]
```diff
@@ -15,16 +15,19 @@ env:
...
 jobs:
   Test-on-k8s-v-1-19-7:
-    runs-on: ubuntu-latest
+    runs-on: self-hosted
     timeout-minutes: 150
     steps:
     - uses: actions/checkout@v2
 
+    - name: Clean up
+      run: kind delete cluster --name chart-testing && docker image prune -f
```
Comment (zryfish) [inline .github/workflows/pr-test.yml:24]: Is the kind cluster name always chart-testing?

Comment (RolandMa1986) [reply .github/workflows/pr-test.yml:24]: For this test, the name was defined in the kind.yaml

File: .github/workflows/kind/kind.yaml (hunk 1 above)
Comment (RolandMa1986) [reply .github/workflows/kind/kind.yaml:9]: Images were pulled inside fo the kind cluster. So we have to mount docker login configuration from host to kind.

File: .github/workflows/pr-test.yml (hunk 2 above)
Comment (RolandMa1986) [reply .github/workflows/pr-test.yml:24]: Yes, it default name. ref: https://github.com/helm/kind-action

**Pull request merged**